invenio_app_ils/permissions.py

```diff
@@ -117,6 +118,14 @@ def loan_extend_circulation_permission(loan):
     return PatronOwnerPermission(loan)
 
 
+def patron_owner_permission(record):
+    """Return permission to allow owner and librarian to access the record."""
+    if not has_request_context():
+        # allows performing the actions out of the request context f.e. CLI
```

Comment:
for other reviewers: please share your thoughts on this solution

currently all of the permissions are blocking us of performing any actions (especially on circulation) programatically, because of `g.identity` (not existing when it is not a request)

Reply:
An alternative artificial workaround would be to use a context when permission is required and manually set the `identity` property - but it's probably anti-pattern.

Reply:
this function is meant to decide to return the permission, so there is no point of adding `@need_permissions`

the point was to check how safe is `has_request_context` as a deciding factor for which permission to return

ping @zzacharo

Reply:
agreed with @zzacharo that it is a solution good enough for now